core/src/main/java/org/apache/iceberg/CatalogUtil.java

```diff
@@ -67,10 +67,18 @@ public class CatalogUtil {
   public static final String ICEBERG_CATALOG_TYPE_HADOOP = "hadoop";
   public static final String ICEBERG_CATALOG_TYPE_HIVE = "hive";
   public static final String ICEBERG_CATALOG_TYPE_REST = "rest";
+  public static final String ICEBERG_CATALOG_TYPE_GLUE = "glue";
```

Comment:
Nice. Can we also update the docs and some testcases to use this short form?

Reply:
+1

Reply:
Sorry, I don't know how to build a glue, nessie and snowflake to test. What should I do?

Reply:
Sorry, I missed this PR. 

Existing test code, you can look up "org.apache.iceberg.nessie.NessieCatalog" and replace few of them with alias. 

And in the docs, we can replace all the occurrences with alias.

Reply:
Let me know if you still want to work on it. If not, I can take it up. Thanks.